src/lib/components/multiDropzone/index.tsx

```diff
@@ -100,38 +101,45 @@ const placeholder = `${textOverrides?.supportsTextShort || "Supports"} ${fileLis
       >
         <input
           data-testid="ds-drop-input"
+          id="ds-drop-input"
```

Comment:
This should be code-generated because there might be more than one instance of the input and HTML Ids should be unique.

You can use our own generateId function:
```
    const [uniqueId] = useState(generateId());
```

Reply:
Of course, makes complete sense that it should be unique. I updated it as you suggested. For now, I left  `data-testid` - once everything works with the label, I can update the tests so they use the label text instead (if it's part of the scope of this ticket).

Reply:
@diogomateus do we need to store the `generateId()` in a `useState` hook?
If it's to prevent this ui to get re-generated on every render, I'm wondering if we can store the variable outside of the component function.
E.g.
```tsx
const multiDropzoneId = generateId();

const MultiDropzone = (props) => {
  ...
  <input
      id={multiDropzoneId}
}
```

Reply:
You're right, the reason is to prevent from re-generating on every render.
Wouldn't storing it outside the component make it always the same for every MultiDropzone import happening?

We could also use `useRef` as it doesn't change unless it's specified.

Reply:
Just a side note: when we upgrade react we could use the [useId function from react ](https://react.dev/reference/react/useId)

Reply:
Ohhh yes, I didn't think about that 😅. Using a `ref` makes more sense.